src/components/timecard/start-finish-time-input/StartFinishTimeInput.jsx

```diff
@@ -14,8 +16,11 @@ const StartFinishTimeInput = ({
   timeEntriesIndex,
   register,
   formState,
+  localStartDate,
+  setLocalEndDate,
 }) => {
   const [errorMessages, setErrorMessages] = useState([]);
+  const [finishTimeText, setFinishTimeText] = useState('');
 
   const desiredErrorOrder = [
```

Comment:
is that still in use ? `sortErrorKeys(errors, desiredErrorOrder).map ...` on line 49 was removed, so maybe that one is required? I may be wrong, but I don't see an application of this array anywhere in the file.